apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1/HiPerfSubscriber.cs

```diff
@@ -115,10 +117,11 @@ public sealed class Settings
         /// <see cref="Reply.Ack"/> or <see cref="Reply.Nack"/> the handled messages.
         /// If the task fails or the handler throws an exception, then the message will be
         /// <see cref="Reply.Nack"/>ed.</param>
+        /// <param name="cancellationToken">Used to abort this operation.</param>
         /// <returns>A task which completes once the message receiver is fully operational.</returns>
         /// <exception cref="InvalidOperationException">
         /// Thrown if this <see cref="HiPerfSubscriber"/> has already been started.</exception>
-        public virtual Task StartAsync(Func<PubsubMessage, Task<Reply>> handler)
+        public virtual Task StartAsync(Func<PubsubMessage, Task<Reply>> handler, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Presumably if the cancellation token is cancelled, that acts like a Stop request, but the final task will be faulted.